common-services.yml

```diff
@@ -21,7 +21,7 @@ services:
       timeout: 5s
 
   logspout:
-    image: audius/logspout:v3.2.14
+    image: audius/logspout:v3.2.14-disable-logging
```

Comment:
This will get created when this PR is merged https://github.com/AudiusProject/audius-protocol/pull/3672

Reply:
This tag looks like it's trying to indicate that this image has logging disabled.

Could we use something like a hash perhaps? (Maybe just off latest master?) Something that doesn't say "disable" in it would be ideal.

The sister PR also looks good, save for this one matching change. 🎉

Reply:
good call. how about we call it like `v3.2.14-update-1`? this also follows some conventional naming standards

Reply:
Nice! I like that direction more!

How about `v3.2.14-1`, which seems more canonical and matches [homebrew](https://formulae.brew.sh/formula/) and some apt packages' version styling?

Reply:
yep sounds good